source/includes/api/_user_management.md

```diff
@@ -293,98 +454,179 @@ curl -X GET http://api.resourcewatch.org/auth/user
 }
 ```
 
+This endpoint allows users with `ADMIN` role to list and filter existing user accounts. Through this endpoint, only 3rd party-based and confirmed email-based user accounts are listed. It's also important to keep in mind that, by default, only users belonging to the same apps as the requesting user will be shown - you can use [filters](#filters337) to modify this behavior.
```

Comment:
Only 3rd party and email based...as opposed to what?

Reply:
As opposed to those pending confirmation via the link sent via email on registration. I'll improve the text, thanks!